main.py

```diff
@@ -1,3 +1,11 @@
 # list of the alphabet (a-z)
-alphabet = [ chr(ord("a") + i) for i in range(26) ]
-reversed_alphabet = alphabet[::-1]
+alphabets = [ chr(ord("a") + i) for i in range(26) ]
+
+# list of the alphabet in reverse
+reversed_alphabets = alphabets[::-1]
+
+# list of numbers (1-100)
+numbers = list(range(101))
+
+# swap 2 variables
+alphabets, numbers = numbers, alphabets
```

Comment:
Please see point 4 in the [README](https://github.com/claesgill/hacktoberfest-oneliners#contribute). Only **1** line each PR.

Reply:
Updated. New commit complies to all rules in the [README](https://github.com/claesgill/hacktoberfest-oneliners#contribute)